clang/lib/StaticAnalyzer/Checkers/DereferenceChecker.cpp

```diff
@@ -31,7 +31,12 @@ class DereferenceChecker
     : public Checker< check::Location,
                       check::Bind,
                       EventDispatcher<ImplicitNullDerefEvent> > {
-  enum DerefKind { NullPointer, UndefinedPointerValue, AddressOfLabel };
+  enum DerefKind {
+    NullPointer,
+    UndefinedPointerValue,
+    AddressOfLabel,
+    FixedAddress
```

Comment:
```suggestion
    FixedAddress,
```
Let's have a trailing comma.